themes/custom/essex/templates/restricted-content/localgov-restricted-content-sign-in-with-message.html.twig

```diff
@@ -1,8 +1,9 @@
 <!--cludooff: index-->
-<div>
-    <p>{{ 'This page is restricted, please sign in to see the full page.'|t }}</p>
-    <div>
-        <a href="/user/login?destination={{ path('<current>') }}">{{ 'Sign in'|t }}</a>
-    </div>
+<div class="ecc-restricted-content">
+  <p class="h3">{{ 'This area is restricted to authorised employees only. Please sign in to access this information.'|t }}</p>
+  <a class="button" href="/user/login?destination={{ path('<current>') }}">
+    {# svg #}
```

Comment:
Is this needed before the PR is ready?

Reply:
@Polynya yeah but not too sure on the best way to add an svg in the template

Reply:
SVG filepath
themes/custom/essex/assets/images/icons/lock.svg

Reply:
Added as an inline element
![image](https://github.com/essexcountycouncil/essex-intranet-drupal/assets/118725785/5a111ad6-3c0c-4d72-903d-dc643b819208)
![image](https://github.com/essexcountycouncil/essex-intranet-drupal/assets/118725785/8d67971e-f3ce-4258-bb7d-57db1ebbbb13)